README.md

```diff
@@ -1,13 +1,23 @@
 # Express API Project
+This weeks project was to start with backend and to build an API using Express. It should have at least two endpoints and be RESTful.
 
-Replace this readme with your own information about your project. 
+## The problem
+It was fun to start with backend and my first feeling is good. Will be fun to learn more!
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+Endpoints I've used:
 
-## The problem
+Root: /
+Songs: /songs (shows the whole data)
+Single song: /songs/id (sorts on specific sonf sorted by id)
+Genre: /genre (sorts on what genre you type after /genre)
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+Tech I've used this week:
+- JS
+- Node.js
+- Express
 
 ## View it live
-
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://johanna-rexin-spotify-popular.herokuapp.com/
+https://johanna-rexin-spotify-popular.herokuapp.com/songs
+https://johanna-rexin-spotify-popular.herokuapp.com/songs/1
+https://johanna-rexin-spotify-popular.herokuapp.com/genre/pop
```

Comment:
Great with some documentation! [Here](https://www.npmjs.com/package/express-list-endpoints)'s a cool lib that you can try out next time if you'd like to include some docs in your API.